DESIGN.md

```diff
@@ -73,77 +157,225 @@ Flow control support:
 - buffering, sampling, throttling, windowing, dropping, etc
 - temporal and count-based strategies
 - `request(n)` consumer demand signal
-  - for pull-based sources, this allows batched "async pull"
-  - for push-based sources, this allows backpressure signals to conditionally apply strategies (i.e. drop, buffer, sample, fail, etc)
+	- for pull-based sources, this allows batched "async pull"
+	- for push-based sources, this allows backpressure signals to conditionally apply strategies (i.e. drop, first, buffer, sample, fail, etc)
+
+You get a flowable from:
+
+- Converting a Observable with a backpressure strategy
+- Create from sync/async OnSubscribe API (which participate in backpressure semantics)
+
+*Type Signature*
+
+```java
+class Flowable<T> implements Flow.Publisher<T>, io.reactivestreams.Publisher<T> {
+  void subscribe(Subscriber<T> subscriber);
+
+  interface Subscriber<T> implements Flow.Subscriber<T>, io.reactivestreams.Subscriber<T> {
```

Comment:
Extending both will prevent direct use of any instance of them directly.

Besides, the signature doesn't compile because it requires

```java
void subscribe(io.reactivestreams.Subscriber)
void subscribe(java.util.concurrent.Flow.Subscriber)
```

to be implemented separately.